packages/rollup-plugin-html/src/RollupPluginHTMLOptions.ts

```diff
@@ -28,6 +28,10 @@ export interface RollupPluginHTMLOptions {
   absoluteBaseUrl?: string;
   /** Whether to set full absolute urls for ['meta[property=og:image]', 'link[rel=canonical]', 'meta[property=og:url]'] or not. Requires a absoluteBaseUrl to be set. Default to true. */
   absoluteSocialMediaUrls?: boolean;
+  /** Should a service worker registration script be injected */
+  injectServiceWorker?: boolean;
+  /** File system path to the generated service worker file */
+  swDest?: string;
```

Comment:
```suggestion
  servicWorkerPath?: string;
```

Seems like a clearer name

Reply:
that's from workbox... it only gets passed on so it's probably good to use the same name here and in the workbox plugin? 🤔

Reply:
True.. through in the workbox plugin I guess there is a bit more context. I guess changing it now in the workbox plugin would be too much

Reply:
rename to `serviceWorkerPath`